examples/config/tests/production.rs

```diff
@@ -0,0 +1,13 @@
+#![feature(plugin)]
+#![plugin(rocket_codegen)]
+
+extern crate rocket;
+use rocket::config::Environment;
+
+mod common;
+use common::test_config;
```

Comment:
Don't use this. Just do `common::test_config` in the `#[test]` function. Same for the rest of the files.